web-client/src/views/FileDocument/ViewAllDocumentsDesktop.jsx

```diff
@@ -42,44 +42,42 @@ export const ViewAllDocumentsDesktop = connect(
-                              onClick={() => {
-                                updateFileDocumentWizardFormValueSequence({
-                                  key: 'category',
-                                  value: doc.category,
-                                });
-                                updateFileDocumentWizardFormValueSequence({
-                                  key: 'documentType',
-                                  value: doc.documentType,
-                                });
-                                updateFileDocumentWizardFormValueSequence({
-                                  key: 'documentTitle',
-                                  value: doc.documentTitle,
-                                });
-                                updateFileDocumentWizardFormValueSequence({
-                                  key: 'eventCode',
-                                  value: doc.eventCode,
-                                });
-                                updateFileDocumentWizardFormValueSequence({
-                                  key: 'scenario',
-                                  value: doc.scenario,
-                                });
-                                openCompleteSelectDocumentTypeModalSequence();
-                              }}
-                            >
-                              {doc.documentType}
-                            </Button>
-                          ),
-                        )}
+                        {viewAllDocumentsHelper.categoryMap[title].map(doc => (
+                          <Button
+                            link
+                            className="text-left"
+                            key={`${title}-document-${doc.eventCode}`}
+                            onClick={() => {
+                              updateFileDocumentWizardFormValueSequence({
+                                key: 'category',
+                                value: doc.category,
+                              });
+                              updateFileDocumentWizardFormValueSequence({
+                                key: 'documentType',
+                                value: doc.documentType,
+                              });
+                              updateFileDocumentWizardFormValueSequence({
+                                key: 'documentTitle',
+                                value: doc.documentTitle,
+                              });
+                              updateFileDocumentWizardFormValueSequence({
+                                key: 'eventCode',
+                                value: doc.eventCode,
+                              });
+                              updateFileDocumentWizardFormValueSequence({
+                                key: 'scenario',
+                                value: doc.scenario,
+                              });
+                              openCompleteSelectDocumentTypeModalSequence();
+                            }}
```

Comment:
This is a little unruly. I know it's not a new change, but could we add a devex card to refactor